### PR TITLE
[stable/redis-sentinel] Add Redis Sentinel chart

### PR DESCRIPTION
**Pull request by KongZ**
This PR adds a new Chart for creating Redis Sentinel cluster.
This chart aims for eliminating complexity of setting up Redis Cluster and using custom images.
You can easily create Redis Sentinel cluster from official Redis image

### CODE REVIEW

File: stable/redis-sentinel/templates/sentinel-statefulset.yaml [hunk 1]
```diff
@@ -0,0 +1,144 @@
+apiVersion: apps/v1beta1
```
Comment (mattfarina) [inline stable/redis-sentinel/templates/sentinel-statefulset.yaml]: Can you consider using `apps/v1beta2` or `apps/v1` for the API version. Both are available in all supported versions of Kubernetes by the community. This applies to all statefulsets in the PR.

**Review by mattfarina — COMMENTED** [review]
The https://github.com/helm/charts/tree/master/stable/redis-ha chart uses sentinel. What does this chart do that shouldn't be part of that one?

File: stable/redis-sentinel/templates/sentinel-statefulset.yaml (hunk 1 above)
Comment (KongZ) [reply stable/redis-sentinel/templates/sentinel-statefulset.yaml]: The `redis-ha` is able to run only on private custom image which image may not available in the future. The master name is also hardcoded in the image which name is `mymaster`.

Comment (mattfarina) [reply stable/redis-sentinel/templates/sentinel-statefulset.yaml]: @KongZ Can you try a pull request to that chart to fix those issues?

**Comment by mattfarina** [comment]
@KongZ I understand there may be issues with the other redis chart. Can or have you tried a pull request to that chart to fix those issues?

**Comment by KongZ** [comment]
@mattfarina that means entire chart will be overridden with one PR. Is this OK?

**Comment by mattfarina** [comment]
@KongZ In the charts repo we try to avoid multiple charts using the same underlying software in the same way. In the near future we will have a setup that enables that but it's different from here.

Can you connect with the maintainers of the other redis chart about changes?

**Comment by cosmicnag** [comment]
+1 for this

**Pull request closed without merging**